.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -55,8 +59,21 @@ body:
 - type: textarea
   attributes:
     label: Actual Results
-    description: >
-      Please paste or describe the results you observe instead of the expected results. If you observe an error, please paste the error message including the **full** traceback of the exception.
+    description: |
+      Please paste or describe the results you observe instead of the expected results. If you observe an error, please paste the error message including the **full traceback** of the exception. For instance for the above code raises the following exception:
```

Comment:
```suggestion
      Please paste or describe the results you observe instead of the expected results. If you observe an error, please paste the error message including the **full traceback** of the exception. For instance the code above raises the following exception:
```